app/models/settings/user_experience.rb

```diff
@@ -21,13 +21,13 @@ class UserExperience < Base
     setting :primary_brand_color_hex, type: :string, default: "#3b49df", validates: {
       format: {
         with: HEX_COLOR_REGEX,
-        message: "must be be a 3 or 6 character hex (starting with #)"
+        message: -> { I18n.t("models.settings.user_experience.message") }
       },
       color_contrast: true
     }
     # a non-public forem will redirect all unauthenticated pages to the registration page.
     # a public forem could have more fine-grained authentication (listings ar private etc.) in future
-    setting :public, type: :boolean, default: true
+    setting :public, type: :boolean, default: 0
```

Comment:
This is a boolean field, so we want `true`, not `0`.

Reply:
This too, will revert.